scripts/override-agent-apm-server.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+# Takes as arguments the apm-server binary path and the apm-server binary file name; 
+# and uncompresses it in the right folder inside the elastic-agent container (must be running)
+
+# Path and file name are separated arguments because the former comes from user input, and the later does not
+# This is because if the file name comes from the user and is not what elastic agent expects, 
+# it will be very hard to debug why it doesn't work
+
+elastic_agent_id=$(docker ps -qf "name=elastic-agent")
+sha=$(docker inspect "$elastic_agent_id" | grep org.label-schema.vcs-ref | awk -F ": \"" '{print $2}' | head -c 6)
+dst=/usr/share/elastic-agent/data/elastic-agent-"$sha"/install/${2%".tar.gz"}
```

Comment:
```suggestion
dst="/usr/share/elastic-agent/data/elastic-agent-${sha}/install/${2}.tar.gz"}
```

Reply:
this is intentional: the folder where to decompress the file is path + filename - "tar.gz" suffix (that is what the `%` is for).

I need to explicitly set the target folder to decompress because otherwise this wont work:
```/bin/bash
~apm-server: mage package
~apm-server: mv apm-server-8.0.0-linux-x86_64.tar.gz apm-server-8.0.0-SNAPSHOT-linux-x86_64.tar.gz
~apm-integration-testing: docker exec <ID> tar zxvf  <PATH-TO>/apm-server-8.0.0-SNAPSHOT-linux-x86_64.tar.gz # will decompress in apm-server-8.0.0-linux-x86_64 and Elastic Agent won't pick it up
```

Reply:
I see it now.